packages/components/data-table-manager/src/data-table-manager.spec.js

```diff
@@ -153,3 +186,97 @@ describe('rendering', () => {
     ).toBeInTheDocument();
   });
 });
+
+describe('rendering with decoupled data table', () => {
```

Comment:
We also want to be sure that when the datatable manager is decoupled, all it's behaviours are still intact.